README.rst

```diff
@@ -40,18 +40,12 @@ How?
   >>> from smart_open import open
   >>>
   >>> # stream lines from an S3 object
-  >>> for line in open('s3://commoncrawl/robots.txt'):
-  ...    print(repr(line))
-  ...    break
+  >>> open('s3://commoncrawl/robots.txt').readline()
```

Comment:
Strong -1: show line iteration in the examples, as prominently as possible.

Why this change?

Reply:
Two reasons:

1. Brevity
2. A for-loop that breaks after the first iteration is redundant

Reply:
We show line iteration in separate examples (e.g. the example below), so there's little value in showing it again here, in my opinion.

Reply:
No, iteration should be the top-most (most visible, most prominent) example.

Also showing it "below" is fine too, to drive the point home.

Reply:
OK, reverted.